assets/src/edit-story/components/library/test/text/textPane.js

```diff
@@ -59,7 +59,9 @@ describe('TextPane', () => {
 
       await getAllFontsPromise;
 
-      fireEvent.click(getByText('Add new text'));
+      fireEvent.click(
+        getByRole('button', { name: 'Add new text', hidden: true })
```

Comment:
`hidden`?

Reply:
It's because the pane is `hidden` by default. Made it active for the purposes of testing: f7bc7cca7